pylint/utils.py

```diff
@@ -841,8 +887,7 @@ def expand_modules(files_or_modules, black_list, black_list_re):
             is_directory = spec.type == modutils.ModuleType.PKG_DIRECTORY
 
         if not is_namespace:
-            result.append({'path': filepath, 'name': modname, 'isarg': True,
-                           'basepath': filepath, 'basename': modname})
+            result.append(ModuleDescription(filepath, modname, True, filepath, modname))
```

Comment:
Can you make isarg a keyword argument, so that thi becomes ModuleDescription(filepath, modname, isarg=True)?